src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -98,16 +98,19 @@ public VerificationResult sendVerificationText(Request req, Response res) {
         OtpUser otpUser = getEntityForId(req, res);
         // Get phone number from the path param.
         String phoneNumber = req.params(PHONE_PARAM);
-        if (StringUtils.isBlank(phoneNumber)) {
-            logMessageAndHalt(req, HttpStatus.BAD_REQUEST_400, "A phone number must be provided.");
-        }
 
-        PhoneNumber validNumber = NotificationUtils.ensureDomesticPhoneNumber(req, phoneNumber);
+        if (!NotificationUtils.isPhoneNumberIsValid(phoneNumber)) {
+            logMessageAndHalt(
+                req,
+                HttpStatus.BAD_REQUEST_400,
+                "Phone number is invalid."
+            );
+        }
 
-        // Update OtpUser.pendingPhoneNumber before submitting SMS request.
-        String phoneNumberToSubmit = validNumber.getPhoneNumber().toString();
-        otpUser.pendingPhoneNumber = phoneNumberToSubmit;
-        otpUser.pendingPhoneNumberFormatted = validNumber.getNationalFormat();
+        // Update OtpUser.phoneNumber before submitting SMS request.
+        String phoneNumberToSubmit = NotificationUtils.getRawPhoneNumber(phoneNumber);
```

Comment:
We should expect clients/requests to contain the country code. Your inclination to make this E.164 format was correct, I'm not sure why you deviated from this. I don't think we really need the config value for country code in this case.

Reply:
I can make the UI add the country prefix if that simplifies the backend. It is odd to ask regular users to enter a +1 every time.

Reply:
Yep, I think that's a more appropriate fix.